src/CreditCard/Loading/Loading.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import '../Loading/Loading.css'
+
+const Loading = () => (
+    <div class="lds-ring">
+        <div></div><div></div><div></div>
```

Comment:
Do you really need those extra `div`s?

Reply:
No, los divs extras son para darle una animacion mejor pero con uno solo funciona 
Dejare solo dos